src/lib/pickles_types/plonk_types.ml

```diff
@@ -827,8 +851,8 @@ module Evals = struct
       ; opt feature_flags.foreign_field_mul
       ; opt feature_flags.xor
       ; opt feature_flags.rot
-      ; opt feature_flags.lookup (* TODO: Fixme *)
-      ; opt feature_flags.lookup (* TODO: Fixme *)
+      ; opt uses_lookup
```

Comment:
It would be nice to comment next to the value what it is corresponding to. It can be infered while reading the code, but the reader (including our future us) takes time to understand.